src/app/productConfig/productConfigModal/productConfig.modal.tpl.html

```diff
@@ -30,7 +30,6 @@
       class="btn btn-md btn-primary btn-block-xs pull-right"
       type="button"
       ng-if="$ctrl.state !== 'errorAlreadyInCart' && $ctrl.state != 'givingLinks'"
-      tabindex="-1"
```

Comment:
The tab order is a little surprising since it goes to Add to Gift Card and then Cancel & Return. Can swap the buttons in the DOM? They UI shouldn't change because of `pull-right`. I think that will make the tab order more intuitive.